LAMP/Login.php

```diff
@@ -7,21 +6,19 @@
 	$firstName = "";
 	$lastName = "";
 
-	$conn = new mysqli("localhost", "TheBeast", "WeLoveCOP4331", "COP4331"); 	
-	if( $conn->connect_error )
-	{
-		returnWithError( $conn->connect_error );
-	}
-	else
+	// include database connection file
+	include_once "dbConfig.php";
+	
+	function loginUser()
 	{
-		$stmt = $conn->prepare("SELECT UserID,firstName,lastName FROM Users WHERE Email=? AND Password =?");
-		$stmt->bind_param("ss", $inData["email"], $inData["password"]);
+		$stmt = $conn->prepare("SELECT UserID,firstName,lastName FROM Users WHERE Login=? AND Password =?");
```

Comment:
In the Users table we actually have It as Email
```suggestion
		$stmt = $conn->prepare("SELECT UserID,firstName,lastName FROM Users WHERE Email=? AND Password =?");
```